lib/tasks/import.rake

```diff
@@ -0,0 +1,167 @@
+namespace :import do
+    desc "data"
+    task :authors => :environment do
+        require 'csv'
+        CSV.foreach('/home/pavan/demo/db/seeders/csv_readers/csv_data/authors-190124.csv',headers: true) do |row|
```

Comment:
In Rails you can access your folder path from _Rails.root_ . This will leads to a static approach i.e. for every system path could be different and you will have to change the code.